ui/res/js/discoveries.js

```diff
@@ -145,14 +145,18 @@ function initDiscoveriesDataTable() {
               ` : ""}
             </tbody>
           </table><div>`;
+          
+          const actions = discoveriesBtnGroupTemplate('Mark as')+`
```

Comment:
actions is not a very good name. We risk to have a conflict because it's too general (and maybe prone to be used for something else)